### PR TITLE
Update falcon7b deployment

### PR DESCRIPTION
**Pull request by shimpa1**
Added conversation history.
TODO: take conversation history into account when generating new responses

### CODE REVIEW

File: Falcon-7B/Dockerfile [hunk 1]
```diff
@@ -2,11 +2,12 @@ from ubuntu:latest
 
 RUN apt update && apt install -y python3 python3-pip
 
-RUN pip install transformers torch einops xformers accelerate
-
+RUN pip install transformers torch einops xformers accelerate falcon
+RUN pip install --upgrade transformers
```
Comment (joshlopes) [inline Falcon-7B/Dockerfile:6]: I don't think you need to do this one, transformers is already being installed above?

File: Falcon-7B/index.html [hunk 2]
```diff
@@ -1,40 +1,44 @@
...
+
       $.ajax({
         url: "/generate",
         type: "POST",
         data: JSON.stringify({ "text": inputText }),
         contentType: "application/json; charset=utf-8",
         dataType: "json",
         success: function(data) {
-          displayGeneratedTexts(data.generated_texts);
+          var response = data.generated_texts[0]; // Assuming you get a single response
```
Comment (joshlopes) [inline Falcon-7B/index.html:30]: and is this a good assumption? should we just loop the array and add them to the conversation?

```javascript
for (const message in data.generated_texts)
    addToConversation('Model: ' + message)
```

File: Falcon-7B/Dockerfile (hunk 1 above)
Comment (shimpa1) [reply Falcon-7B/Dockerfile:6]: There used to be an error in earlier testing versions when it was not upgraded, this line remained and is indeed unnecessary, thank you!

File: Falcon-7B/index.html (hunk 2 above)
Comment (Dimokus88) [reply Falcon-7B/index.html:30]: Hey @shimpa1 !
You can look to the recommendation @joshlopes ?

Comment (shimpa1) [reply Falcon-7B/index.html:30]: hey guys!
Thanks for reminding me, let me try it out!

**Comment by Dimokus88** [comment]
Ready to merge @andy108369

File: Falcon-7B/index.html (hunk 2 above)
Comment (andy108369) [reply Falcon-7B/index.html:30]: @shimpa1 status please :)

**Comment by andy108369** [comment]
> Ready to merge @andy108369

Thanks!

Waiting for https://github.com/akash-network/awesome-akash/pull/449#discussion_r1475256693

File: Falcon-7B/index.html (hunk 2 above)
Comment (Dimokus88) [reply Falcon-7B/index.html:30]: Hey @shimpa1 !
up status =)

**Comment by shimpa1** [comment]
hey guys,
I can't really dedicate enough time for this and I'm making you all wait and waste time. Sorry about that.
Please feel free to contribute in any way you think would be beneficial and administer the project to your liking.
thank you!
Shimpa

**Pull request closed without merging**